src/cli/cli.ts

```diff
@@ -0,0 +1,19 @@
+import { Command } from 'commander';
+
+let program : Command;
+
+export const initCLI = () => {
+    program = new Command();
+    program
+    .option('-f, --filepath <string>', "A path to load a kamon game from a file")
+    .parse(process.argv);
+}
+
+export const getFilePath = ():string => {
+    const filePathParameter = program.opts()['filepath'];
+
+    return filePathParameter != null ? filePathParameter : `${__dirname}/../../save.json`
```

Comment:
Path traversal vulnerability here. Use `path.join()` to create a composite path from user input (and `path.normalize()` to avoid problems in the path).

https://www.stackhawk.com/blog/node-js-path-traversal-guide-examples-and-prevention/

Reply:
It just replaced it, thanks for the link.